Feature/skeleton component

jhossyd3v opened the pull request:
- Create Skeleton component
- Build section with skeleton on Hypophosphatasia page

Commit:
feat: :bento: Add basic files for Skeleton component

Add basic files for Skeleton component, astro file, scss file and main image

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/skeleton-component

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/skeleton-component

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/skeleton-component

Commit:
feat: :lipstick: Add image to skeleton and basic styles

Add image inside the skeleton container and basic styles for general container

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/skeleton-component

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/skeleton-component

Commit:
style: :lipstick: Add basic HTML structure and basic styles

Add main HTML elements and basic styles, add some styles for active elements

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/skeleton-component

Commit:
feat: :construction: Complete mobile skeleton component

Add styles for mobile devices, change name of component, add script file for component

Commit:
feat: :construction: Add file for desktop skeleton component

Copy file from Mobile Skeleton and change required data

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/skeleton-component

Commit:
style: :lipstick: Add styles for desktop version and animation

Add styles for Skeleton desktop component, change font-sizes, line-heights and active line directions, add keyframe animation to show/hide the list

Commit:
refactor: :recycle: Fix script skeleton for desktop version

Update code and fix functions for skeleton component working on desktop version in a correct way

\

Commit:
fix: :bug: Fix bug with the fade out animation

Fix on style and js a bug with the fade out animation, add the animation to the hidden class for the list and remove a setTimeout on the script

Commit:
feat: :construction: Add HTML elements and styles for skeleton section

Create the section with texts and skeleton component on Hypophosphatasia Page

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/skeleton-component

HackMort reviewed (changes requested):
Overall looks great! A couple of tweaks:
1. Can we make the font on mobile a bit smaller? Looks too crowed in my opinion.
2. Renal: on mobile the line is too short and the Word should be right aligned.
3. This symbol should be up, use <sup> symbol </sup> and adjust as necesary.
![image](https://user-images.githubusercontent.com/6466646/232612233-759e0cb7-55a6-47c1-89d4-cb553e96613f.png)
4. By default the "skeleton__point" does not have a border and on "active" it does, this create a little jump/move to the word. Add a transparent border by default that should fix it.
5. Skeletal element does not have enough space for the content box, see screenshot:
![image](https://user-images.githubusercontent.com/6466646/232612643-ff8dacfe-ee65-4f2d-8514-29ca9c497af5.png)
6. Make the animation speed a bit faster.
7. "Hypophosphatasia compromises more than good bones" Heading should be only on 2 lines on desktop.

Commit:
style: :lipstick: Add transparent border to skeleton__point

Add transparent border to skeleton__point when point is not active

Commit:
fix: :bug: Fix word alignment and long of a line

Fix long line of renal item on mobile,  fix the alignment of renal word on mobile and all the words on desktop

Commit:
fix: :bug: Fix height of symptoms list

Fix the height of symptoms list that was too short for long list

Commit:
style: :lipstick: Change the speed of animation

Change speeds for fade in and fade out animations for symptoms list

Commit:
style: :lipstick: Fix heading for desktop

Fix the heading of the section for desktop, remove padding-left from content section

Commit:
style: :lipstick: Add an HTML element for symbol

Add sup tag HTML for symbol and change the top position according to design

Commit:
style: :lipstick: Reduce the font-size of skeleton words on mobile

Reduce font-size from 25px to 21px for skeleton words on mobile

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into feature/skeleton-component

HackMort reviewed (approved):
Awesome work!
